### ISSUE TITLE
Context has incorrect headers

### ISSUE THREAD

**Issue by martinsondergaard** [Pseudonium/Obsidian_to_Anki#130]
When adding context to cards the header structure is incorrect. 

The context being added to my card is `Hooke's Law.md > Standard version > Materials version`
Both `Standard version` and `Material version` are headers at the same level; in this case header level 3.

I would expect the context to be `Hooke's Law.md > Materials version`

Here is the file:

```markdown
Hooke's Law is a [[Constitutive Equation]] that relates the [[Elastic and Plastic deformation|Elastic]] deformation of some object to the external force applied to it. 

---

### Standard version

The law is commonly explained in terms of a spring under some force. The force applied to the spring $F_S$ and the extension of the spring $x$ are related as follows: 
$$F_S = kx$$
The spring constant $k$ is a property of the spring and is dependent on material properties as well as the structure of the spring.

> **Warning**: Sometimes Hooke's Law includes a minus sign in which case the force $F_S$ no longer represents the applied external force, but instead represents the internal reaction force.

---

### Materials version

A different version is seen in [[Mechanics of Materials]], where the [[Mechanical Stress|Stress]] $\sigma$ and [[Strain]] $\epsilon$ are related as follows:
$$\sigma = E \epsilon$$
Here $E$ is the material equivalent of the spring constant and is called the [[Young's modulus]].



---

START
Cloze
Text: In mechanics of materials the equation for {Hooke's Law} is: {\[\sigma=E\epsilon\]}
Extra: where \(\sigma\) is the stress in the material, \(E\) is the Young's modulus of the material, and \(\epsilon\) is the resulting strain.
<!--ID: 1607771572295-->
END
```

**Comment by Pseudonium** [comment]
Right, I coded it assuming it’d go heading level 1, level 2 etc.

I’ll see if I can change the context algorithm to account for this!

**Comment by Pseudonium** [comment]
I also notice that you're using Anki math formatting for the cards - you allowed to use the `$x = 5$` and `$$z = 10$$` syntax! The script will auto-convert it.

**Comment by Pseudonium** [comment]
Alright, I've fixed this for v3.2.0

**Comment by martinsondergaard** [comment]
Didn't know about the conversion for math, that's great! I thought it would force LaTeX instead of Mathjax, but much easier if I can just use dollar signs :)

**Comment by Pseudonium** [comment]
Released 3.2.0, should fix this issue
